packages/client/src/layers/network/createNetworkLayer.ts

```diff
@@ -28,6 +28,9 @@ export async function createNetworkLayer(config: GameConfig) {
     LoadingState: defineLoadingStateComponent(world),
     Position: defineCoordComponent(world, { id: "Position", metadata: { contractId: "component.Position" } }),
     CarriedBy: defineStringComponent(world, { id: "CarriedBy", metadata: { contractId: "component.CarriedBy" } }),
+    Name: defineStringComponent(world, { id: "Name", metadata: { contractId: "component.Name" } }),
+    Authors: defineStringComponent(world, { id: "Authors", metadata: { contractId: "component.Authors" } }),
```

Comment:
This should not be `defineStringComponent`

Reply:
There is no equivalent for arrays in the [official](https://github.com/latticexyz/mud/tree/main/packages/std-client/src/components) library. In order to solve this quickly, I'm going to convert the AuthorsComponent into a string instead of an array of strings.